src/py4dgeo/util.py

```diff
@@ -254,6 +265,28 @@ def is_iterable(obj):
     return isinstance(obj, collections.abc.Iterable) and not isinstance(obj, str)
 
 
+def initialize_openmp_defaults():
+    """Set OpenMP environment variables for optimal performance on Windows with llvm OpenMP"""
+
+    # Only apply when using Windows
+    if platform.system() != "Windows":
+        return
+
+    # Only set if the user hasn't already
+    if "OMP_NUM_THREADS" not in os.environ:
+        try:
+            import psutil
+
+            num_cores = psutil.cpu_count(logical=False)
+            os.environ["OMP_NUM_THREADS"] = str(num_cores)
+        except Exception:
```

Comment:
Try to always specify the exception type you are expecting to catch - otherwise you are opening yourself up to weird errors: A typo in the try block might cause e.g. a `NameError` and as a result you would silently set the number of threads to something else. In this particular case `ImportError` would be an option, but I am not convinced this is a scenario we need to handle - we properly install `psutil` after all. If `psutil.cpu_count` is known to throw under certain circumstances, identify the exception type and catch that. If we do not know of such circumstances, we might as well remove the try/except block.